frontend/src/components/GMainToolbar.vue

```diff
@@ -115,171 +105,136 @@ SPDX-License-Identifier: Apache-2.0
       icon
       class="mr-3"
     >
-      <v-tooltip
-        :disabled="menu"
-        location="left"
+      <v-badge
+        v-if="isAdmin"
+        color="primary"
+        location="bottom right"
+        icon="mdi-account-supervisor"
       >
-        <template #activator="{ props }">
-          <v-badge
+        <v-avatar
+          v-tooltip="{ text: isAdmin ? `${avatarTitle} Operator` : avatarTitle, location: 'left', disabled: menu }"
```

Comment:
styling on tooltip on avatar is lost

Reply:
Reverted avatar tooltip (due to various issues)